src/query/sql/src/planner/binder/ddl/column.rs

```diff
@@ -24,6 +30,9 @@ use crate::BindContext;
 use crate::Binder;
 use crate::SelectBuilder;
 
+pub const GLOBAL_PRIV: &str = "0x5f3759df";
```

Comment:
Please don't use constant value here, we should xor the permission as the final value.